sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberTreeTest.kt

```diff
@@ -158,6 +160,9 @@ class SentryTimberTreeTest {
     }
 
     @Test
+    @Ignore(
```

Comment:
Should we make it public then? Just a getter?
Alternatively for testing purposes we could use reflection

Reply:
I meant it's package-private in timber, we can't really get it, aside from reflection. So we have no way to expose it either. :) It's not only for testing, but also for end users - now when they do `Timber.tag("test"); Timber.e("issue")` we have no way to access this tag (that's also what I was talking about in the PR comments thread)